charts/cla/templates/secrets.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: github-credentials
+  namespace: {{ .Values.namespace }}
+  labels:
+    name: github credentials
+  annotations:
```

Comment:
I think this is not useful, annotations should be used for declaratively identifying the resource for other tools to use it.